ios/FluentUI/Table View/TableViewCellTokenSet.swift

```diff
@@ -81,10 +81,16 @@ public class TableViewCellTokenSet: ControlTokenSet<TableViewCellTokenSet.Tokens
         super.init { token, theme in
             switch token {
             case .backgroundColor:
-                return .dynamicColor { theme.aliasTokens.colors[.background1] }
+                return .dynamicColor {
+                    DynamicColor(light: theme.aliasTokens.colors[.background1].light,
```

Comment:
is there an intended usage of background1 to still use the elevated colors?

Reply:
The following controls use BG1:

- `BadgeField`
- `CardNudge`
- `Notification` (primary outline)
- `PersonaButtonCarousel`
- `PersonaListView`
- `SegmentedControl`

So far only `SegmentedControl` explicitly ensures it doesn't use elevated colors. I don't know how to determine if any of these controls 'need' elevated colors.